Bug fix install

fdintino opened the pull request:
I've included a bug fix to the ansible task config for elasticsearch and a correction to the order of operations for installation in the README. The specifics for each are in their respective commit messages.

davidmcclure commented:
Hey @fdintino,

Thanks for this! Was that Elasticsearch role erroring out? I just checked it on a fresh VM, and it went through OK. What OS / version of Ansible are you on?

fdintino commented:
I ran this on my development machine, which is currently on OS X 10.9.5. I have ansible 1.9.3 installed. Here was the output of running the command without those changes:

```
$ ansible-playbook deploy-workers.yml

PLAY [Deploy a worker set] ****************************************************

GATHERING FACTS ***************************************************************
ok: [server]

TASK: [common | Install generic dependencies] *********************************
changed: [server] => (item=git,htop)

TASK: [common | Mount the OSP volume] *****************************************
changed: [server]

TASK: [redis | Add the Redis PPA] *********************************************
changed: [server]

TASK: [redis | Install Redis] *************************************************
changed: [server]

TASK: [redis | Apply the configuration] ***************************************
changed: [server]

TASK: [redis | Check that Redis is running] ***********************************
ok: [server]

TASK: [elasticsearch | Stop Elasticsearch] ************************************
failed: [server] => {"failed": true}
msg: no service or tool found for: elasticsearch

FATAL: all hosts have already failed -- aborting

PLAY RECAP ********************************************************************
           to retry, use: --limit @/Users/fdintino/deploy-workers.retry

server                     : ok=7    changed=5    unreachable=0    failed=1
```

davidmcclure commented:
Ah, I think that's an Ansible 1.x vs 2.0 difference. These playbooks assume 2.0, which is pegged in the [requirements.txt](https://github.com/davidmcclure/osp-deploy/blob/master/requirements.txt#L5) file. The easiest thing is just to create a fresh virtualenv and use that:

```
virtualenv env
. env/bin/activate
pip install -r requirements.txt
```

I updated the docs, and left in place the sequencing change in the Elasticsearch role, since it makes more sense anyway to install the package first.

Thanks,
David
